lib/fog/ovirt/models/compute/server.rb

```diff
@@ -31,6 +31,8 @@ class Server < Fog::Compute::Server
         attribute :ips
         attribute :ha
         attribute :ha_priority
+        attribute :clone
```

Comment:
Alternatively, we could define clone_attributes as methods, that would do the mapping between how one would want to define the clone attributes, and how rbovirt maps it (as the way the rbovirt does it is not very nice).

```
def clone_attributes
  extract_clone_attributes(@attributes)
end

# Accepted keys: :clone, :disks
def clone_attributes=(clone_attributes)
  @attributes.merge!(extract_clone_attributes(clone_attributes)
end

def extract_clone_attributes(attributes)
  ret = {}
  ret[:clone] = attributes[:clone] if attributes.key?(:clone)
  ret[:disks] = attributes[:disks] if attributes.key?(:disks)
  ret
end
```

And in the vm options, we would pass `:clone_attributes => { :clone => true, :disks => [...]}`